notify/Makefile

```diff
@@ -3,7 +3,8 @@
 # make licensed : will build a license limited version of the plugin
 #
 
-FILES = bin/ etc/ share/
+#FILES = bin/ etc/ share/
```

Comment:
why?

Reply:
There are no files to put in the etc/ folder, which will result in a failure at compilation time.
Some files comes directly from a postinst execution and not from a copy while installing the plugin.